pom.xml

```diff
@@ -137,6 +137,7 @@
     <lombok.version>1.18.20</lombok.version>
     <brotli-codec.version>0.1.1</brotli-codec.version>
     <aircompressor.version>0.20</aircompressor.version>
+    <iceberg.version>0.12.0</iceberg.version>
```

Comment:
theres a new iceberg release https://github.com/apache/iceberg/releases/tag/apache-iceberg-0.12.1

Maybe we should use it directly?

Reply:
Yep, thanks for pointing to this. When the PR was created, only 0.12.0 was present, but since the newer version was released, we can use that one.